go-eicio/event.go

```diff
@@ -42,6 +36,34 @@ func (evt *Event) AddCollection(collection Message, name string) {
 		collHdr.Type = EventHeader_CollectionHeader_MCParticle
 	case *SimTrackerHitCollection:
 		collHdr.Type = EventHeader_CollectionHeader_SimTrackerHit
+	case *TrackerRawDataCollection:
```

Comment:
Thanks for your feedback, it is welcome and highly valued!  I agree that this should be done differently, and your suggestion is one I'll consider.